src/basic/control.lean

```diff
@@ -83,11 +83,6 @@ instance monad_except.of_statet {ε σ} {m : Type → Type} [monad m] [monad_exc
 , catch := λ α S e, ⟨λ s, catch (state_t.run S s) (λ x, state_t.run (e x) s)⟩ -- [note] alternatively, you could propagate the state.
 }
 
-instance monad_except.alt {ε} [inhabited ε] {m : Type → Type} [monad m] [monad_except ε m] : alternative m :=
```

Comment:
(This looks unused, so I just deleted it rather than fixing it, but let me know if that's incorrect)